src/applications/static-pages/sidebar-navigation.js

```diff
@@ -31,15 +31,17 @@ class SideBarMenu {
 
   openMenu(trigger) {
     this.getMenu(trigger).classList.add('va-sidebarnav--opened');
-    document.body.classList.add('va-pos-fixed');
+    document.getElementsByClassName('va-btn-sidebarnav-trigger')[0].setAttribute('hidden',  'true');
+    document.getElementsByTagName('body')[0].style.overflow = 'hidden';
     this.closeMenu(trigger);
   }
 
   closeMenu() {
     const close = this.menu.querySelector('.va-sidebarnav-close');
     close.addEventListener('click', () => {
       this.menu.classList.remove('va-sidebarnav--opened');
-      document.body.classList.remove('va-pos-fixed');
+      document.getElementsByTagName('body')[0].style.overflow = 'initial';
+      document.getElementsByClassName('va-btn-sidebarnav-trigger')[0].removeAttribute('hidden');
```

Comment:
Made a ticket to double check this change, as it does affect Vets.gov - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13517